requests/models.py

```diff
@@ -657,6 +657,9 @@ def iter_content(self, chunk_size=1, decode_unicode=False):
         read into memory.  This is not necessarily the length of each item
         returned as decoding can take place.
 
+        chunk_size must be of type int or None. A value of None implies
+        reading data in chunks of an undefined/inconsistent size.
+
```

Comment:
Now that we're here, we may as well be clear about exactly what this does. It'll take quite a few more LoC, but it'll help us by being a better explanation.